kernel/xpmem_private.h

```diff
@@ -268,7 +268,7 @@ struct xpmem_partition {
  */
 #define XPMEM_MAX_PAGE_FAULT_AFTER 32
 #define XPMEM_MAX_PAGE_FAULT_BEFORE 16
-#define XPMEM_MAX_PAGE_FAULTS 128
+#define XPMEM_MAX_PAGE_FAULTS 96
```

Comment:
why needed?

Reply:
This is to reduce stack usage. An array of pointers of pages to map (`struct page`) is used when handling faults on the stack. Reducing to 96 as it's already beyond any observed gain.

- https://github.com/openucx/xpmem/blob/0de4990fa160f479194fc98ecb4f33889499c952/kernel/xpmem_attach.c#L196